### ISSUE TITLE
New Lines is config cause new Lines in api response 

### ISSUE THREAD

**Issue by CovertError** [laravel/framework#52646]
### Laravel Version

10.48.20

### PHP Version

8.2

### Description

the issue is if you add any empty lines in any config file you will see those empty lines in the api response for some reason 

check this repo

https://github.com/CovertError/LivewireTest

basically there is one api http://livewiretest.test/api/test

if you send a get request you will get a ton of empty lines because there are empty lines in the config file app.php but this is not just for the app.php for any config file 

### Steps To Reproduce

check this repo

https://github.com/CovertError/LivewireTest

basically there is one api http://livewiretest.test/api/test

if you send a get request you will get a ton of empty lines because there are empty lines in the config file app.php but this is not just for the app.php for any config file

**Comment by CovertError** [comment]
this has nothing to do with livewire just a test project

**Comment by noefleury** [comment]
The explanation is here :

https://github.com/laravel/framework/blob/769f00ba71de3b3cbbb271e9f34a019b584982c9/src/Illuminate/Foundation/Bootstrap/LoadConfiguration.php#L96

From [PHP manual](https://www.php.net/manual/en/function.include.php) :

> The include expression includes and evaluates the specified file.

So I think it cannot be considered as an issue as you are explicitly manipulating some PHP source files.

**Comment by CovertError** [comment]
I was told by Tylor "That’s how PHP works? Those lines are not within PHP tags and will be sent to the browser"
I guess this can be closed
